lib/extensions/ar_slug.rb

```diff
@@ -0,0 +1,13 @@
+module ArSlug
+  extend ActiveSupport::Concern
+
+  included do
+    virtual_column :href_slug, :type => :string
```

Comment:
haven't discussed that.  I'm game to changing it if that's desired,  is there a preference to ArHrefSlug maybe ?  @Fryguy ?